lang/it/validation.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => "L'attributo :attribute deve essere accettato.",
```

Comment:
You should escape a single quote. like: L'attributo should be L\'attributo.

Reply:
I replaced the ' to " for the string, so the escape for L'attributo is not needed.

Am I wrong?
I also did the same above.

Reply:
ok @Sanjaybhattwebkul 
please verify the changes after testing.

Reply:
@vivek-webkul so is it ok?